Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde_json = "1.0.105"
 
 # crates needed for the auto splitter management code to work as expected`
 anyhow = { version = "1.0.75", optional = true }
+livesplit-auto-splitting = { git = "https://github.com/LiveSplit/livesplit-core", optional = true }
```

Comment:
It makes no sense for this to be optional... or to even have that dependency at all. The functions we need probably should be re-exported in livesplit-core, such that it is all we need.

Reply:
Okay I've added https://github.com/LiveSplit/livesplit-core/pull/758 to re-export what I need in `livesplit-core`.

Reply:
And I've switched the import to use `livesplit_core::auto_splitting` instead of `livesplit_auto_splitting`.